code/modules/error_handler/error_handler.dm

```diff
@@ -154,3 +177,9 @@ GLOBAL_VAR_INIT(total_runtimes_skipped, 0)
 #endif
 
 #undef ERROR_USEFUL_LEN
+
+/// Exists to trigger infinite recursion runtimes in testing
+/proc/recurse(times)
+	if(times <= 0)
+		return
+	recurse(times - 1)
```

Comment:
Would it be dumb to have a unit test ensure this is caught correctly

Reply:
no but I couldn't decide how to actually test for that, cause we want to look at the log file right

Reply:
update 
<img width="677" height="330" alt="image" src="https://github.com/user-attachments/assets/7b6a7cf4-fb3d-4bca-85ce-ffb0f6ca1d68" />